tests/test_helpers/test_profiling.py

```diff
@@ -29,24 +29,39 @@
 from tests.common.utils import wait_for_condition
 
 
+class DummyClass:
+    """Dummy class for counting purposes"""
+
+    def __init__(self) -> None:
+        """Dummy init"""
+        pass
```

Comment:
leave it, but not needed since it inherits the `__init__` frem `object` implicitly